draft-ietf-quic-manageability.md

```diff
@@ -476,26 +482,26 @@ therefore not part of the wire image that is visible to on-path observers.
 
 ## Version Negotiation and Greasing {#version}
 
-Version negotiation is not protected, given the used protection mechanism can
-change with the version.  However, the choices provided in the list of version
-in the Version Negotiation packet will be validated as soon as the cryptographic
-context has been established. Therefore any manipulation of this list will be
-detected and will cause the endpoints to terminate the connection.
+Version negotiation packets are not intrinsically protected, but QUIC versions
+can use later encrypted messages to verify that they were authentic.
+Therefore any manipulation of this list will be detected and will cause the
+endpoints to terminate the connection.
```

Comment:
Not sure if this is the reaction is correct.  "The connection" isn't a notion that makes sense when VN is involved because there are multiple connection attempts involved: one that elicited the VN, and one that used the VN.  Neither will result in a connection being established if there was tampering.

Maybe "might terminate the connection attempt".